### PR TITLE
embed update + typos

### PR DESCRIPTION
**Pull request by kseirru**
• added support for specifying an author in the Embed
• fix some typos
________________________________

Check List
- [x] add support for specifying an author 
- [x] add image support
- [x] add thumbnail support

### CODE REVIEW

**Comment by bleudev** [comment]
Пж челик пиши на английском)))

**Comment by kseirru** [comment]
ok

**Comment by bleudev** [comment]
And please name your commits and pulls on english)

File: disspy/application_commands.py [hunk 1]
```diff
@@ -82,18 +81,18 @@ async def execute(url, payload, token):
         Returns:
             _type_: _description_
         """
-        async with aiohttp.ClientSession(
-            headers={'Authorization': f'Bot {token}', 'content-type': 'application/json'}) as s:
+        async with aiohttp.ClientSession(headers={'Authorization': f'Bot {token}', 'content-type': 'application/json'}) as s:
```
Comment (bleudev) [inline disspy/application_commands.py:84]: Why??? PyLint say that this line is so big

Comment (kseirru) [reply disspy/application_commands.py:84]: ¯\_(ツ)_/¯

**Comment by bleudev** [comment]
Please add your checklist in description of pull

File: disspy/application_commands.py (hunk 1 above)
Comment (bleudev) [reply disspy/application_commands.py:84]: Please fix it again

**Comment by bleudev** [comment]
This pull still in progress? Or ready? (I'm ready for merge)

**Comment by kseirru** [comment]
i think ready,

**Comment by bleudev** [comment]
Ok i'm merge

**Pull request merged**